src/posh-git.psd1

```diff
@@ -1,10 +1,10 @@
 @{
 
 # Script module or binary module file associated with this manifest.
-ModuleToProcess = 'posh-git.psm1'
+RootModule = 'posh-git.psm1'
 
 # Version number of this module.
-ModuleVersion = '0.7.0'
+ModuleVersion = '1.0.0'
```

Comment:
What if we leave in `Write-Prompt` to abstract the difference between using Write-Host and ANSI escape sequences.  Folks customizing their own prompt function might find that useful.